runner/ansible/roles/1.1.2.runtime/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Set Test ID
+  set_fact:
+    test_id: '1.1.2_runtime'
```

Comment:
I think we should have here `1.1.2.runtime` (see the `.` instead of the `_`), to have the same id of the role folder.
This should be changed in all the runtimes (including 1.1.1.runtime, which I didn't find in the previous review)